app/src/main/java/com/ttbmp/cinehub/app/service/email/MockEmailService.java

```diff
@@ -8,6 +8,6 @@
 
     @Override
     public void sendMail(EmailServiceRequest emailServiceRequest) {
-        System.out.println("Email sent");
+        emailServiceRequest.setEmail("Payment with success");
```

Comment:
This is a generic method to send any kind of emails, this implementation doesn't really make sense.